holoviews/plotting/bokeh/element.py

```diff
@@ -105,6 +105,10 @@ class ElementPlot(BokehPlot, GenericElementPlot):
     align = param.ObjectSelector(default='start', objects=['start', 'center', 'end'], doc="""
         Alignment (vertical or horizontal) of the plot in a layout.""")
 
+    apply_hard_bounds = param.Boolean(default=True, doc="""
```

Comment:
Not 100% sure about changing the default, I think it may be sensible but I'm not 100% sure. Particularly if we default to applying the combined ranges, e.g. if the x\ylim or the Dimension.range is smaller than the data range what would even be the point of sending that additional data that you can't even see?

Reply:
defaulting to False now